embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/JdbcInputConnection.java

```diff
@@ -187,35 +190,69 @@ public String buildSelectQuery(String tableName,
         sb.append(selectColumnList.or("*"));
         sb.append(" FROM ").append(buildTableName(actualTableName));
 
-        if (whereCondition.isPresent()) {
-            sb.append(" WHERE ").append(whereCondition.get());
+        if (whereCondition.isPresent() || incrementalColumn.isPresent()) {
+            sb.append(" WHERE ");
+            if (incrementalColumn.isPresent() && lastRecord.isPresent()) {
+                for (int i = 0; i < incrementalColumn.get().size(); i++) {
+                    String column = incrementalColumn.get().get(i);
+                    sb.append(column).append( " > \"");
+                    sb.append(lastRecord.get().get(column).replace("\"", "\\\""));
```

Comment:
Escaping quoting character with `\` is invalid as a standard SQL (it's a MySQL extension). It doesn't work with PostgreSQL, for example.